sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -520,16 +532,13 @@ private bool TryGetReturnTypePropertyBindings(ITypeSymbol returnTypeSymbol, bool
                 return true;
             }
 
-            private bool HasHttpResponseattribute(IPropertySymbol prop)
+            private bool HasHttpResultAttribute(ISymbol prop)
```

Comment:
This method is still checking for the first attribute of the symbol. Would it work for something like below, which is valid code customer may write.

```csharp
[SuppressMessage("Microsoft.Naming", "Foo", Justification = "Bar")]
[HttpResult]
public string Api_Response { set; get; }
```